.travis.yml

```diff
@@ -38,7 +38,7 @@ matrix:
     - php: 7.2
       env: STABILITY=dev SYMFONY_VERSION=4.0.*
     - php: 7.1
-      env: STABILITY=dev COMPOSER_FLAGS="--prefer-lowest" SYMFONY_VERSION=2.8.* SYMFONY_DEPRECATIONS_HELPER=weak
+      env: STABILITY=dev COMPOSER_FLAGS="--prefer-lowest" SYMFONY_VERSION=^2.8.18 SYMFONY_DEPRECATIONS_HELPER=weak
```

Comment:
Because of the `--prefer-lowest` Composer flag being used, we need to specify the Symfony version which introduced PHPUnit 6 support.

Reply:
This change must be reflected in the `dev-kit` template.

Reply:
should we just change composer.json of testing to say it needs at least symfony 2.8.18?